packages/core/src/database/seed.ts

```diff
@@ -84,7 +86,7 @@ export const createDatabaseCli = (dsn: string) => {
       pool.query(insertInto(managementResource, 'resources')),
       pool.query(insertInto(createDefaultSetting(), 'settings')),
       pool.query(insertInto(defaultSignInExperience, 'sign_in_experiences')),
-      pool.query(insertInto(createDemoAppApplication(), 'applications')),
+      pool.query(insertInto(createDemoAppApplication(buildApplicationSecret()), 'applications')),
```

Comment:
Could move the generator within the createDemoAppApplication method?

Reply:
demo app is a SPA, so it's fine to leave it empty?

Reply:
I set `secret` to non-nullable, will generate secret to all type of applications.